ldirectord/ldirectord.in

```diff
@@ -153,6 +153,23 @@ but ONLY if using forking mode (B<fork = >I<yes>).
 
 Default: 10 seconds
 
+B<checkintervalspread = >I<n>
+
+Defines the maximum number of second to be randomly added to checkinterval.
+
+When fork=no this option is not used.
+
+When fork=yes this option defines the maximum additional amount of time
+each forked child sleeps per virtual service pool before running all
+realserver checks for that pool.
+
+If set in the virtual server section then the global value is overridden.
+
+This option causes checks to spread out in time, in order to avoid
+overloading services probed using the same test.
```

Comment:
Abstracting checks from services might be nicer, but that would be much more difficult to do.